R/GroupSequentialDesign.R

```diff
@@ -0,0 +1,183 @@
+#' Group-sequential design
+#'
+#' [ToDo]
+#'
+#' @slot n1 stage-one sample size
+#' @slot c1f early stopping for futility boundary
+#' @slot c1e early stopping for efficacy boundary
+#' @slot n2 stage-two sample size
+#' @slot c2_pivots vector of length order giving the values of c2 at the
+#'     pivot points of the numeric integration rule [TODO: these are not available during construction]
+#' @slot x1_norm_pivots normalized pivots for integration rule (in [-1, 1])
+#' @slot weights weights of conditional score values at x1_norm_pivots for
+#'     approximating the integral over x1.
+#'
+#' @exportClass GSDesign
+setClass("GSDesign", representation(
+        n1        = "numeric",
+        c1f       = "numeric",
+        c1e       = "numeric",
+        n2        = "numeric",
```

Comment:
strictly this is n2_continue or something - otherwise could be confused with our n2 function, right? I.e. why is design@n2 different from n2(design, x1) in some cases ;)